src/System.ComponentModel.TypeConverter/src/System/Timers/TimersDescriptionAttribute.cs

```diff
@@ -37,6 +36,11 @@ public override string Description
                 if (!_replaced)
                 {
                     _replaced = true;
+                    if (base.Description == null)
+                    {
+                        throw new ArgumentNullException("name");
+                    }
+
```

Comment:
Why was this added?

Reply:
Reverted - the exception param name is different between netfx and netcore